### PR TITLE
CR-277: Change log to debug calls, log additional metadata

### PR DESCRIPTION
**Pull request by seth-riedel**

### CODE REVIEW

File: redisinsight/api/src/common/logger/app-logger.ts [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+    for (const k in optionalParams) {
+      if (this.isException(optionalParams[k])) {
+        errorMessage = (optionalParams[k] as Error).message;
+        errorStack = (optionalParams[k] as Error).stack;
+      } else if (optionalParams[k] !== contextArg) {
+        meta.push(optionalParams[k]);
+      }
+    }
+    const metaObj = meta.length > 1 ? meta : meta[0];
```
Comment (ArtemHoruzhenko) [inline redisinsight/api/src/common/logger/app-logger.ts:67]: what if `meta.length === 0`?

File: redisinsight/api/src/common/logger/app-logger.ts [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        : {}),
+    };
+  }
+
+  /**
+   * Write a 'log' level log.
+   */
+  log(message: string | LogObject, ...optionalParams: ErrorOrMeta[]) {
+    const parsedArgs = this.parseLoggerArgs(message, optionalParams);
+    if (!AppLogger.startupContexts.includes(parsedArgs.context)) {
```
Comment (ArtemHoruzhenko) [inline redisinsight/api/src/common/logger/app-logger.ts:87]: DO we want to ignore nestjs core logs?

Comment (seth-riedel) [reply redisinsight/api/src/common/logger/app-logger.ts:87]: I think so. this hides all the starup logs where Nest logs that it is registering all the routers, etc. I don't think this suppresses any other logs that we care about. What do you think?

**Comment by ArtemHoruzhenko** [comment]
@seth-riedel Looks good. Will wait for `clone` + tests

**Comment by seth-riedel** [comment]
@ArtemHoruzhenko please check when you can

**Review by ArtemHoruzhenko — APPROVED** [review]

**Pull request merged**